### PR TITLE
Add safe_mode Discord message

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

File: data/discord/messages/homeassistant.yaml [hunk 1]
```diff
@@ -1375,3 +1375,19 @@ chatgpt:
...
+safe_mode:
+  description: "Enable safe mode"
+  content: >-
+    You can restart your Home Assistant instance in safe mode.
+    This will disable the loading of anything custom, like custom integrations, custom cards, custom frontend modules, and custom themes.
+
+    This is useful to enable you to rule out potential issues caused by running custom _things_.
+
+    You can enable safe mode by going to [Settings -> System](<https://my.home-assistant.io/redirect/system_dashboard/>)
+    in your Home Assistant UI, then clicking the power button in the top right corner,
```
Comment (joostlek) [inline data/discord/messages/homeassistant.yaml:1388]:
```suggestion
    in your Home Assistant UI, then selecting the power button in the top right corner,
```

**Review by gjohansson-ST — APPROVED** [review]

File: data/discord/messages/homeassistant.yaml (hunk 1 above)
Comment (ludeeus) [reply data/discord/messages/homeassistant.yaml:1388]:
```suggestion
    in your Home Assistant UI, then hitting the power button in the top right corner,
```

Comment (ludeeus) [reply data/discord/messages/homeassistant.yaml:1388]:
```suggestion
    in your Home Assistant UI, then touching the power button in the top right corner,
```

Comment (ludeeus) [reply data/discord/messages/homeassistant.yaml:1388]:
```suggestion
    in your Home Assistant UI, then using the power button in the top right corner,
```

Comment (ludeeus) [reply data/discord/messages/homeassistant.yaml:1388]: 🤷‍♂️

Comment (joostlek) [reply data/discord/messages/homeassistant.yaml:1388]: I mean, I am now being taught by Frenck and Coffeecat that we should use device unrelated wording in docs so why not here as well 🤷🏻

File: data/discord/messages/homeassistant.yaml [hunk 2]
```diff
@@ -1375,3 +1375,19 @@ chatgpt:
...
+
+safe_mode:
+  description: "Enable safe mode"
+  content: >-
+    You can restart your Home Assistant instance in safe mode.
+    This will disable the loading of anything custom, like custom integrations, custom cards, custom frontend modules, and custom themes.
+
+    This is useful to enable you to rule out potential issues caused by running custom _things_.
+
+    You can enable safe mode by going to [Settings -> System](<https://my.home-assistant.io/redirect/system_dashboard/>)
```
Comment (ludeeus) [inline data/discord/messages/homeassistant.yaml:1387]:
```suggestion
    You can enable safe mode by going to [**Settings** > **System**](<https://my.home-assistant.io/redirect/system_dashboard/>)
```

File: data/discord/messages/homeassistant.yaml [hunk 3]
```diff
@@ -1375,3 +1375,19 @@ chatgpt:
...
+    You can restart your Home Assistant instance in safe mode.
+    This will disable the loading of anything custom, like custom integrations, custom cards, custom frontend modules, and custom themes.
+
+    This is useful to enable you to rule out potential issues caused by running custom _things_.
+
+    You can enable safe mode by going to [**Settings** > **System**](<https://my.home-assistant.io/redirect/system_dashboard/>)
+    in your Home Assistant UI, then selecting the power button in the top right corner,
+    this will open the restart dialog.
+
+    Under "Advanced options", you will find the "Restart Home Assistant in safe mode" option.
```
Comment (ludeeus) [inline data/discord/messages/homeassistant.yaml:1391]:
```suggestion
    Under **Advanced options**, you will find the **Restart Home Assistant in safe mode** option.
```

**Review by c0ffeeca7 — APPROVED** [review]
LGTM, @ludeeus :+1:

**Review by joostlek — APPROVED** [review]

**Pull request merged**